### PR TITLE
Time Scarcity blog template

### PR DESCRIPTION
**Pull request by bhumika-gupta**
Completed blog template for Next on self-sabotage: time scarcity article.

### CODE REVIEW

File: _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: Next on Self-Sabotage::&ensp;Time Scarcity
```
Comment (xsharonhe) [inline _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx]: change this to ```title: 'Next on Self-Sabotage: Time Scarcity'```. apparently that is how you add colons in markdown for front matter :0

File: _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+<p> A lot of people struggle with this, so you don’t need to feel alone! 
+The first step to combatting this problem is becoming aware of it, which I just helped you overcome 🙂 
+After the first step, it becomes more comfortable as you come to terms with it and manage it appropriately. </p>
+<p> As J. R. R. Tolkien says, “All we have to decide is what to do with the time that is given to us.” </p>
+
+<h3> More resources if you’re interested in reading more about this! </h3>
+
+<ul>
+    <li> <a href="http://runrunlive.com/time-scarcity-versus-time-abundance" target="_blank" rel="noopener noreferrer"><Time Scarcity versus Time Abundance></a></li>
+    <li> <a href="hhttps://www.highlights.lornerubis.com/2018/06/the-daily-fight-between-abundance-and-scarcity/" target="_blank" rel="noopener noreferrer"><The Daily Fight Between Abundance and Scarcity></a></li>
```
Comment (xsharonhe) [inline _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx]: my template is probably a poor example but it should be:

```
<li> <a href="http://runrunlive.com/time-scarcity-versus-time-abundance" target="_blank" rel="noopener noreferrer">Time Scarcity versus Time Abundance</a></li>
    <li> <a href="hhttps://www.highlights.lornerubis.com/2018/06/the-daily-fight-between-abundance-and-scarcity/" target="_blank" rel="noopener noreferrer">The Daily Fight Between Abundance and Scarcity</a></li>

```
    
anything in tags will need a closing tag so we don't ever use <> in html

File: _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+or a snack or taking a short walk around my house to maintain my energy levels. </p>
+
+<p> Ensure you are not using your break time going on social media or reading your emails as 
+this can produce unnecessary feelings of stress and may cause you to work on another task. 
+Then, It’s quite likely that you will get into that loop of feeling like there’s not enough time 
+for the things you want to do, all over again. </p>
+
+<p> When we are immersed in this bubble of spending our break time online, we feel the need to compete. </p>
+
+<h3> Final words 🙂 </h3>
```
Comment (xsharonhe) [inline _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx]: id remove the emojis because they don't always work and same on line 133

**Review by xsharonhe — COMMENTED** [review]
Small changes :)

File: _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx (hunk 2 above)
Comment (bhumika-gupta) [reply _digest/january-2021/next-on-self-sabotage-time-scarcity.mdx]: yeah i was a bit confused bc in your example u put <> but sounds good

**Review by xsharonhe — APPROVED** [review]
LGTM

**Pull request merged**